The attrs package includes type annotations or type stubs since version 18.2.0,
so we do not need types-attrs.

--- astroid-2.15.7/requirements_test_brain.txt.orig
+++ astroid-2.15.7/requirements_test_brain.txt
@@ -1,5 +1,4 @@
 attrs
-types-attrs
 nose
 numpy>=1.17.0; python_version<"3.11"
 python-dateutil
